### PR TITLE
Fix some UI bug on test module

### PR DESCRIPTION
**Pull request by wolf427**
1. The service test page will throw undefined exception in console if the "service" parameter in url is null.
2. Keep the style of service test page consistent with other pages

### CODE REVIEW

File: dubbo-admin-ui/src/components/test/ServiceTest.vue [hunk 1]
```diff
@@ -230,7 +237,7 @@
     mounted () {
       this.$store.dispatch('loadServiceItems')
       let query = this.$route.query
-      this.filter = query['service']
+      this.filter = query['service'] ? query['service'] : ''
```
Comment (kezhenxu94) [inline dubbo-admin-ui/src/components/test/ServiceTest.vue]: `query['service'] ? query['service'] : ''` can be replaced with a simpler way `query['service'] || ''`

**Comment by wolf427** [comment]
Before:
before
Now:
now

**Comment by wolf427** [comment]
@kezhenxu94 
THX for your advice

**Comment by DLive** [comment]
LGTM.
It passed the test locally.
Thanks for you contribution.

**Pull request merged**